### DISCUSSION TITLE
Why are errors reported when processing PBF files.

### DISCUSSION THREAD

**Discussion by mshmoon** [Universite-Gustave-Eiffel/NoiseModelling#568] [Q&A]
![image](https://user-images.githubusercontent.com/38131350/230535017-81a1f6ca-990d-4f2c-939a-b49abf8576a3.png)
I downloaded some city PBF files from the geofabrik website, but noisemadelling reported errors when processing most of the files

**Comment by gpetit** [comment]
Hi,
You should open an issue (since it's not really a discussion).
Without additional info (like logs, ...) it's hard to say, but it seems that your input files have encoding errors (missing text values ?)
